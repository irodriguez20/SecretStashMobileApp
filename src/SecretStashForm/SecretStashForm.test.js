import React from "react";
import SecetStashForm from "./SecretStashForm";
import toJson from "enzyme-to-json";
import { shallow } from "enzyme";

describe(`SecretStashForm component`, () => {
    const props = {
        className: "test-class-name",
        children: <p>test children</p>,
        "data-other": "test-other-prop"
    };

    it("renders a form.SecretStashForm by default", () => {
        const wrapper = shallow(<SecetStashForm />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    it("renders the SecretStashForm given props", () => {
        const wrapper = shallow(<SecetStashForm {...props} />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
