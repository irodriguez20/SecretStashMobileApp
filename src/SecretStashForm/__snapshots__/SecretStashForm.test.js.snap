// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecretStashForm component renders a form.SecretStashForm by default 1`] = `
<form
  action="#"
  className="SecretStash-form "
/>
`;

exports[`SecretStashForm component renders the SecretStashForm given props 1`] = `
<form
  action="#"
  className="SecretStash-form test-class-name"
  data-other="test-other-prop"
>
  <p>
    test children
  </p>
</form>
`;
